gfx.cpp

```diff
@@ -104,6 +104,7 @@ class GfxInternal
     ID3D12GraphicsCommandList6 *mesh_command_list_ = nullptr;
     ID3D12CommandAllocator **command_allocators_ = nullptr;
     std::vector<IAmdExtD3DDevice1 *> amd_ext_devices_;
+    ID3D12DebugCommandList *dbgCommandList = nullptr;
```

Comment:
Only used for debugging that resource states match

Reply:
Small thing but naming convention is snake case with trailing underscore in this codebase 🙂

Reply:
fixed